### PR TITLE
chore: update eslint config and fix issues

### PR DESCRIPTION
**Pull request by imurchie**

### CODE REVIEW

File: lib/commands/performance.js [hunk 1]
```diff
@@ -129,8 +129,8 @@ commands.mobileStartPerfRecord = async function mobileStartPerfRecord (opts = {}
   });
   proc.on('output', (stdout, stderr) => {
     (stdout || stderr).split('\n')
-      .filter(x => x.length)
-      .map(x => log.debug(`[instruments] ${x}`));
+      .filter((x) => x.length)
```
Comment (mykola-mokhnach) [inline lib/commands/performance.js]: Please don't change that, so we don't get a merge conflict

Comment (imurchie) [reply lib/commands/performance.js]: Sure. I won't merge until your PR is in.

**Comment by mykola-mokhnach** [comment]
https://travis-ci.org/github/appium/appium-xcuitest-driver/jobs/662092796#L7869 - it looks like something got broken after the recent wd update

**Comment by imurchie** [comment]
@mykola-mokhnach I don't think it is a `wd` update. The problem is the "elements" returned from `appium-webdriveragent`. 

What we used to get, for that test:
```
dbug WD Proxy Proxying [POST /elements] to [POST http://127.0.0.1:8100/session/5F9222C0-9980-4F1D-992D-9FC1EE390901/elements] with body: {"using":"class name","value":"XCUIElementTypeButton"}
dbug WD Proxy Got response with status 200: {"value":[{"ELEMENT":"00000000-0000-0000-0000-000000000000","element-6066-11e4-a52e-4f735466cecf":"00000000-0000-0000-0000-000000000000"},{"ELEMENT":"00000000-0000-0000-0000-000000000000","element-6066-11e4-a52e-4f735466cecf":"00000000-0000-0000-0000-000000000000"},{"ELEMENT":"00000000-0000-0000-0000-000000000000","element-6066-11e4-a52e-4f735466cecf":"00000000-0000-0000-0000-000000000000"},{"ELEMENT":"00000000-0000-0000-0000-000000000000","element-6066-11e4-a52e-4f735466cecf":"00000000-0000-0000-0000-000000000000"},{"ELEMENT":"00000000-0000-0000-0000-000000000000","element-6066-11e4-a52e-4f735466cecf":"00000000-0000-0000-0000-000000000000"},{"ELEMENT":"00000000-0000-0000-0000-000000000000","element-6066-11e4-a52e-4f735466cecf":"00000000-0000-0000-0000-000000000000"},{"ELEMENT":"00000000-0000-0000-0000-000000000000","element-6066-11e4-a52e-4f735466cecf":"00000000-0000-0000-0000-000000000000"},{"ELEMENT":"00000000-0000-0000-0000-000000000000","element-6066-11e4-a52e-4f735466cecf":"00000000-0000-0000-0000-00...
dbug MJSONWP (d3f2a9ab) Responding to client with driver.findElements() result: [{"element-6066-11e4-a52e-4f735466cecf":"00000000-0000-0000-0000-000000000000","ELEMENT":"00000000-0000-0000-0000-000000000000"},{"element-6066-11e4-a52e-4f735466cecf":"00000000-0000-0000-0000-000000000000","ELEMENT":"00000000-0000-0000-0000-000000000000"},{"element-6066-11e4-a52e-4f735466cecf":"00000000-0000-0000-0000-000000000000","ELEMENT":"00000000-0000-0000-0000-000000000000"},{"element-6066-11e4-a52e-4f735466cecf":"00000000-0000-0000-0000-000000000000","ELEMENT":"00000000-0000-0000-0000-000000000000"},{"element-6066-11e4-a52e-4f735466cecf":"00000000-0000-0000-0000-000000000000","ELEMENT":"00000000-0000-0000-0000-000000000000"},{"element-6066-11e4-a52e-4f735466cecf":"00000000-0000-0000-0000-000000000000","ELEMENT":"00000000-0000-0000-0000-000000000000"},{"element-6066-11e4-a52e-4f735466cecf":"00000000-0000-0000-0000-000000000000","ELEMENT":"00000000-0000-0000-0000-000000000000"},{"element-6066-11e4-a52e-4f735466cecf":"00000000-0000-0000-0000-000000000000","ELEMENT":"00000000-0000-0000-0000-00000000000...
```
What we get now, which causes the errors:
```
dbug [09-54-24:825] WD Proxy Proxying [POST /elements] to [POST http://127.0.0.1:8100/session/C12DF3AE-E1A7-4807-94AB-50D426A6944B/elements] with body: {"using":"class name","value":"XCUIElementTypeButton"}
dbug [09-55-07:649] WD Proxy Got response with status 200: {"value":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{"ELEMENT":"AF000000-0000-0000-3F1A-010000000000","element-6066-11e4-a52e-4f735466cecf":"AF000000-0000-0000-3F1A-010000000000"},{"ELEMENT":"B4000000-0000-0000-3F1A-010000000000","element-6066-11e4-a52e-4f735466cecf":"B4000000-0000-0000-3F1A-010000000000"},{"ELEMENT":"B9000000-0000-0000-3F1A-010000000000","element-6066-11e4-a52e-4f735466cecf":"B9000000-0000-0000-3F1A-010000000000"},{"ELEMENT":"BE000000-0000-0000-3F1A-010000000000","element-6066-11e4-a52e-4f735466cecf":"BE000000-0000-0000-3F1A-010000000000"},{"ELEMENT":"C3000000-0000-0000-3F1A-010000000000","element-6066-11e4-a52e-4f735466cecf":"C3000000-0000-0000-3F1A-010000000000"},{"ELEMENT":"C8000000-0000-0000-3F1A-010000000000","element-6066-11e4-a52e-4f735466cecf":"C8000000-0000-0000-3F1A-010000000000"},{"ELEMENT":"CD000000-0000-0000-3F1A-010000000000","element-6066-11e4-a52e-4f735466cecf":"CD000000-0000-0000-3F1A-010000000000"}],"sessionId":"C12DF3AE-E1A7-4807-94AB-50D426A6944B"}
dbug [09-55-07:649] MJSONWP (65a609c7) Responding to client with driver.findElements() result: [{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{"element-6066-11e4-a52e-4f735466cecf":"AF000000-0000-0000-3F1A-010000000000","ELEMENT":"AF000000-0000-0000-3F1A-010000000000"},{"element-6066-11e4-a52e-4f735466cecf":"B4000000-0000-0000-3F1A-010000000000","ELEMENT":"B4000000-0000-0000-3F1A-010000000000"},{"element-6066-11e4-a52e-4f735466cecf":"B9000000-0000-0000-3F1A-010000000000","ELEMENT":"B9000000-0000-0000-3F1A-010000000000"},{"element-6066-11e4-a52e-4f735466cecf":"BE000000-0000-0000-3F1A-010000000000","ELEMENT":"BE000000-0000-0000-3F1A-010000000000"},{"element-6066-11e4-a52e-4f735466cecf":"C3000000-0000-0000-3F1A-010000000000","ELEMENT":"C3000000-0000-0000-3F1A-010000000000"},{"element-6066-11e4-a52e-4f735466cecf":"C8000000-0000-0000-3F1A-010000000000","ELEMENT":"C8000000-0000-0000-3F1A-010000000000"},{"element-6066-11e4-a52e-4f735466cecf":"CD000000-0000-0000-3F1A-010000000000","ELEMENT":"CD000000-0000-0000-3F1A-010000000000"}]
```
I have no idea why the empty element objects are being returned.

**Pull request merged**